contrib/galaxy_supervisor.conf

```diff
@@ -38,7 +37,7 @@ autostart       = true
 autorestart     = true
 startsecs       = 20
 user            = galaxy
-environment     = PYTHONHOME=/home/galaxy/galaxy/.venv
+environment     = PATH=/home/galaxy/galaxy/.venv/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin,VIRTUAL_ENV=/home/galaxy/galaxy/.venv
```

Comment:
The job handlers should only need:
```
VIRTUAL_ENV=/home/galaxy/galaxy/.venv
```

Reply:
Actually, they shouldn't need anything at all.

Reply:
Are you sure about that? I found a problem with the virtualenv not activating causing the metadata setting to fail, this is why I added the VIRTUAL_ENV setting.

Reply:
I can confirm (as of 18.09) that `VIRTUAL_ENV` is no longer needed.